net-fs/samba/files/samba-4.16.1-netdb-defines.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/nsswitch/wins.c b/nsswitch/wins.c
```

Comment:
1. Run `iwdevtools` scrub-patch on this file
2. Add link to any relevant Gentoo bugs to the top of the patch (and tag them in the commit)